pyEpiabm/pyEpiabm/tests/test_routine/test_space_forces.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.property.infection_status import InfectionStatus
```

Comment:
```suggestion
from pyEpiabm.property import InfectionStatus
```

May need to add to __init__.py in thei subpackage if this doesn't work yet, but should be consistent with the other import!